haskell/src/Indigo/Contracts/Safelist.hs

```diff
@@ -16,6 +16,7 @@ import qualified Data.Set as Set
 
 import Indigo as Indigo
 import Lorentz.Run (Contract)
+import Prelude ()
```

Comment:
Erm, did it become necessary?

Reply:
@awkure ping.

Reply:
Yes, because it conflicts with `import Indigo` recently updated operators

Reply:
Alright, maybe it would be better to enable `NoImplicitPrelude` globally for the lib, but I am not sure, so let's not do it for now.